opal/util/show_help.c

```diff
@@ -58,13 +59,28 @@ static int opal_show_help_internal(const char *filename, const char *topic, int
                                    ...);
 static void opal_show_help_finalize(void);
 
+typedef struct {
+    pmix_info_t *info;
+    pmix_info_t *dirs;
+    char *msg;
+} opal_log_info_t;
+
 opal_show_help_fn_t opal_show_help = opal_show_help_internal;
 opal_show_vhelp_fn_t opal_show_vhelp = opal_show_vhelp_internal;
 
 int opal_show_help_init(void)
 {
     opal_output_stream_t lds;
 
+    opal_help_want_aggregate = true;
+    mca_base_var_register("opal", NULL, "base", "help_aggregate",
```

Comment:
We should either make an MCA alias for `orte_base_help_aggregate` or add this to the list of things that are different between v4.x and v5.x.

Reply:
Good point, I'll add an alias.

Reply:
I think I'm only 33% in favor of an alias.  I'd be willing to bet that few -- if any -- users actually used `orte_base_help_aggregate`.  It might be ok to just document the change.

Reply:
Added it to the list.